tests/integration/targets/ios_bgp_global/tests/cli/merged.yaml

```diff
@@ -27,6 +28,14 @@
               remote_as: 100
               shutdown:
                 set: false
+            - neighbor_address: 192.0.2.1
+              remote_as: "500.65083"
+              local_as:
+                number: "501.65083"
```

Comment:
This should be an integer in the test, lets make sure that people using this can continue to use the the same way they were using before.